schemas/os.yml

```diff
@@ -24,7 +24,14 @@
       type: keyword
       example: "Mac OS X"
       description: >
-        Operating system name.
+        Operating system name, without the version.
+
+    - name: full_name
```

Comment:
Is that really the `full_name`. This looks to strongly inspired by `user.full_name` but I don't think it's the same. Otherwise it would be `ruflin 18` ;-) 

Do we need this in ECS?

Reply:
Open to suggestions for other names. Could be `os.description`, perhaps. Or just `os.full`. I want the concept, but I have no strong attachment to the name `full_name`, I agree it's not perfect.

Reply:
Changed to `.full`. I'm good with that.

I think `full_name` was a bit more clear on what it was, but I can see your point vs `user.full_name`. There's some patterns we can use in many places (like the customizable `.name`), but I wouldn't have put `.full_name` in those patterns, so I didn't really see a conflict there :-)